feat: created ONBOARDING.md doc. (#117)
mParticle/mparticle-android-sample-apps: core-sdk-samples/higgs-shop-sample-app/ONBOARDING.md

6. On the project-level settings.gradle file include the kit using:  
   `include ':mparticle-android-integration-`*replace with kit name*
   `project(':mparticle-android-integration-*replace with kit name*').projectDir = new File('app/libs/mparticle-android-integration-`*replace with kit name*)

7. Finally add the sample app credentials on the main class cdHiggsShopSampleApplication() replacing .credentials(*your apiKey,*your apiSecret*)

#### A good way to test kit integration is by placing a breakpoint inside the kit's onCreate() and run the app on debug mode, the breakpoint should trigger indicating the kit is being initialized.

# Adding the SDK to a local mParticle sample app project.

Do to the complexity of the multi-module configuration of the SDK adding it directly as an operable module in the sample app is not currently supported, different steps must be taken to *build the SDK locally and add it as a dependency.*

1. First fork a local copy of the SDK repository, in this local copy you must make *all the configuration changes that you wish to see implemented on your local sample app* before you make a local build.

> 2. Once your changes are made run the following command  `./gradlew publishReleaseLocal`  this will create a local snapshot of the SDK.

Reviewer: ```suggestion
2. After your changes are made, run the command  `./gradlew publishReleaseLocal`.  This command creates a local snapshot of the SDK.
```

> 3. Finally in the sample app's *module-level build.gradle.kts* file replace the original core implementation with yours, you can find the name in your local **m2** folder it should contain the word "SNAPSHOT" in it, the complete dependency should look like this: `implementation("com.mparticle:android-core:#.##.#-SNAPSHOT"`

Reviewer: ```suggestion
3. Finally, in the sample app's *module-level build.gradle.kts* file, replace the original core implementation with yours. You can find the name in your local **m2** folder. The dependency should contain the word "SNAPSHOT." The complete dependency should be similar to the following: `implementation("com.mparticle:android-core:#.##.#-SNAPSHOT"`.
```
